test/wdt_e2e_simple_test.sh

```diff
@@ -137,7 +137,13 @@ if [ $DO_VERIFY -eq 1 ] ; then
     echo "Should be no diff"
     (cd $DIR; diff -u src.md5s dst.md5s)
     STATUS=$?
+    if [ $STATUS -eq 0 ] ; then
+      (cd $DIR/src ; ( find . -type f -printf %f%m\\n | sort ) > ../src.perm )
```

Comment:
`find . -type f -printf %f%m\\n` doesn't work on Mac.  Can you also test files with permissions other than 644?